testnet/CLI/Commands/orders/index.md

```diff
@@ -17,6 +17,8 @@ spctl orders <command>
 |**Command**|**Description**|
 | :- | :- |
 |[cancel](/testnet/cli/commands/orders/cancel)|Cancel order with `id`|
+|[download-result](/testnet/cli/commands/orders/download-result)|Download result of order with `id`|
 |[get](/testnet/cli/commands/orders/get)|Display detailed information on order with `id`|
 |[list](/testnet/cli/commands/orders/list)|List orders|
-|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
\ No newline at end of file
+|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
+|[withdraw-deposit](/testnet/cli/commands/orders/download-result)|Withdraw unspent deposit from a completed order with `id`|
```

Comment:
I think the path should be ending with the name of the command itself and not "download-result" in this case - "/testnet/cli/commands/orders/withdraw-deposit"
And what's the difference between commands "replenish" and "withdraw" here? To me, they've similar meaning